pom.xml

```diff
@@ -181,6 +182,9 @@
                 <artifactId>cobertura-maven-plugin</artifactId>
                 <version>2.7</version>
                 <configuration>
+                    <check>
```

Comment:
Is this necessary?

Reply:
According to IntelliJ, yes. Without the check-tag, the configuration tag is shown in red.